nilearn/plotting/surf_plotting.py

```diff
@@ -1469,14 +1469,27 @@ def plot_img_on_surf(stat_map, surf_mesh='fsaverage5', mask_img=None,
 
 
 @fill_doc
-def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
-                  hemi='left', view='lateral', engine='matplotlib',
-                  threshold=1e-14, alpha='auto', vmin=None, vmax=None,
-                  cmap='gist_ncar', cbar_tick_format="auto",
-                  bg_on_data=False, darkness=.7,
-                  title=None, title_font_size=18,
-                  output_file=None, axes=None,
-                  figure=None, **kwargs):
+def plot_surf_roi(surf_mesh,
+                  roi_map,
+                  bg_map=None,
+                  hemi='left',
+                  view='lateral',
+                  engine='matplotlib',
+                  avg_method='median',
```

Comment:
Added a default argument that was implicitly passed to to `plot_surf` (but with a different default than that of plot_surf) and could lead to similar problems as for `symmetric_cmap`.

This is technically a change of the API in the doc though it could always be passed as kwarg. Should we mention it in an "added in version" ? I would say no but this is not a hill I will die on.

Reply:
Maybe not indeed (no strong feeling on my side.)